fda-model/compute/simulation.cs

```diff
@@ -52,7 +61,7 @@ internal Simulation()
         /// <param name="iterations"></param>
         /// <param name="computeDefaultThreshold"></param>
         /// <returns></returns>
-        public Results Compute(interfaces.IProvideRandomNumbers rp, Int64 iterations, bool computeDefaultThreshold = true){
+        public Results Compute(interfaces.IProvideRandomNumbers rp, Int64 iterations, bool computeDefaultThreshold = true, bool giveMeADamageFrequency = false){
```

Comment:
i think we can do like i did with the ead message, just send a damage frequency message - if there are no listeners, the message wont fire.

Reply:
lets do it!